contrib/devtools/lint-locale-dependence.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+LOCALE_DEPENDENT_FUNCTIONS=(
+    isalnum isalpha isblank iscntrl isdigit isgraph islower isprint ispunct
+    isspace isupper iswalnum iswalpha iswblank iswcntrl iswctype iswdigit
+    iswgraph iswlower iswprint iswpunct iswspace iswupper iswxdigit isxdigit
+    stoi stol stoll strcoll strftime strptime strtod strtol strtoll tolower
+    toupper towlower towupper
+)
+REGEXP_IGNORE_EXTERNAL="src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/)"
+REGEXP_IGNORE_KNOWN_VIOLATIONS="^(src/base58.cpp:.*isspace|src/qt/rpcconsole.cpp:.*isdigit|src/rest.cpp:.*strtol|src/torcontrol.cpp:.*strtol|src/uint256.cpp:.*isspace|src/uint256.cpp:.*tolower|src/util.cpp:.*tolower|src/utilmoneystr.cpp:.*isdigit|src/utilmoneystr.cpp:.*isspace|src/utilstrencodings.cpp:.*isspace|src/utilstrencodings.cpp:.*strtoll|src/utilstrencodings.cpp:.*strtol)"
```

Comment:
That's a shocking number of known violations.
I didn't know e.g. `strtol` is also locale dependent.

Reply:
Is it possible to split this list over multiple lines, so that it's possible to remove an exception by removing a line?